site/content/en/docs/Tasks/configmaps_and_secrets.md

```diff
@@ -1,8 +1,51 @@
 ---
 title: "ConfigMaps and Secrets"
 linkTitle: "ConfigMaps and Secrets"
-weight: 3
+weight: 1
 date: 2017-01-05
 description: >
```

Comment:
I suggest updating the dates to September 2023. 

Of course, I think we need a clear guideline on updating the dates.

Reply:
I updated this to the current date for all task files.